notifications_utils/broadcast_areas/__init__.py

```diff
@@ -98,25 +77,19 @@ def unenclosed_polygons(self):
         ]
 
 
-class BroadcastAreaLibrary(SerialisedModelCollection, IdFromNameMixin, GetItemByIdMixin):
+class BroadcastAreaLibrary(SerialisedModelCollection, IdentifiableMixin, IdFromNameMixin, GetItemByIdMixin):
 
     model = BroadcastArea
 
-    def __init__(self, filename):
-        self.name, geojson_data = load_geojson_file(filename)
-        self.items = geojson_data['features']
-
-    def get_examples(self, max_displayed=4):
+    def __init__(self, library_name):
+        self.name = library_name
 
-        truncate_at = max_displayed - 1
+    def get_examples(self):
```

Comment:
Where/how do we use this get_examples method? 
I understand that how it shows an example of the dataset for a given library like "countries". But when the dataset is large like it is for local authorities how will it be useful?

Reply:
Currently this is used [when selecting a library](https://github.com/alphagov/notifications-admin/blob/78bda9d2679faad1f4611866816ef154da304e54/app/templates/views/broadcast/libraries.html#L180)

ie on this page 
![image](https://user-images.githubusercontent.com/1482692/88664389-e7debd80-d0d4-11ea-85cc-8a74c7f72b51.png)

with the changes currently in this PR it looks something like

```
Choose where to broadcast to

Counties and Unitary Authorities in England and Wales
Barking and Dagenham, Barnet, Barnsley, 171 more…

Countries
England, Northern Ireland, Scotland and Wales

Regions of England
East Midlands, East of England, London, 6 more…

Electoral Wards of the United Kingdom
Eden - Ravenstonedale, Folkestone and Hythe - Sandgate & West Folkestone, East Lothian - Musselburgh, Daventry - Weedon, 8591 more…
```